defaults/main.yml

```diff
@@ -17,6 +17,7 @@ clone_vm: DebianBusterTemplate    # The VM to clone
 clone_storage: local-lvm          # Target storage for full clone.
 clone_format: raw                 # Target drive's backing file's data format. Used only with clone
 clone_target: "{{ node }}"        # Target node
+clone_timeout: 500                # Cloning timeout, can take a while
```

Comment:
As I mentioned in the PR conversation, I think it's more convenient to define a variable (eg `pve_timeout`) of general purpose for both `tasks/clone_kvm.yml` and `tasks/create_kvm.yml`

What do you think?